### PR TITLE
Enable Use of More Info Page

### LINKED ISSUE #37: Create More Info Component
**Is your feature request related to a problem? Please describe.**
We do not have a component to display more information about an article yet. 

**Describe the solution you'd like**
Create a component and render it on the more info component.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Got the more info page working 

## Purpose
Enable users to learn more about an article

## Approach
1. Created a MoreInfo component
2. Created a selectedArticle state
3. Created a class to implement an interface for the selectedArticle
4. Created an interface for props for the MoreInfo article

Closes #37

### CODE REVIEW

File: the-toad-tribune/src/api/newsApi.ts [hunk 1]
```diff
@@ -24,7 +29,21 @@ export interface Article {
...
+    id: "",
+    name: "",
+  };
+  author = "";
+  title = "";
+  description = "";
+  url = "";
+  urlToImage = "";
+  publishedAt = "";
+  content = "";
```
Comment (JorgeLVilla) [inline the-toad-tribune/src/api/newsApi.ts:46]: Noice

**Review by JorgeLVilla — APPROVED** [review]
Nice job Kenny!

**Review by connielion — APPROVED** [review]
Looks great! Keep up the good work.

File: the-toad-tribune/src/layout/MoreInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { IMoreInfoPageProps } from "../api/newsApi";
+import styled from "styled-components";
+
+const MoreInfo: React.FC<IMoreInfoPageProps> = ({ selectedArticle }) => {
+    console.log(selectedArticle);
```
Comment (Gjhnsn) [inline the-toad-tribune/src/layout/MoreInfo.tsx:5]: still need the console log?

Comment (kmausolf) [inline the-toad-tribune/src/layout/MoreInfo.tsx:5]: Console log here.

Comment (krferrell) [reply the-toad-tribune/src/layout/MoreInfo.tsx:5]: Thanks

**Review by Gjhnsn — APPROVED** [review]
Everything looks good, had 1 console log in MoreInfo.tsx

**Review by dannysanchez559 — APPROVED** [review]
looking good

**Pull request merged**